app-frontend/src/app/index.routes.js

```diff
@@ -300,28 +301,42 @@ function labStates($stateProvider) {
             url: '/lab',
             redirectTo: 'lab.browse'
         })
-        .state('lab.build', {
-            title: 'Build a Tool',
-            url: '/build/:toolid',
+        // later on we'll use this to view / edit user templates
+        .state('lab.template', {
+            title: 'View a Template',
+            url: '/template/:templateid',
             parent: 'lab',
-            templateUrl: labTpl,
-            controller: 'LabController',
-            controllerAs: '$ctrl',
-            redirectTo: 'lab.build.run'
+            templateUrl: labTemplateTpl,
+            controller: 'LabTemplateController',
+            controllerAs: '$ctrl'
+        })
+        .state('lab.createtool', {
```

Comment:
I think we should hyphenate multi-word routes...`create-tool` in this case. It could also just be `create` or `new`.

Reply:
I'll change it to create-tool, since we will probably want a create-template route in the future